src/main/java/core/basesyntax/Item.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Item {
```

Comment:
Not the example of the perfect naming :)
```suggestion
public interface AreaCalculator {
```